python/pyspark/sql/streaming/query.py

```diff
@@ -330,7 +335,8 @@ def stop(self) -> None:
         Stop streaming query
 
         >>> sq.stop()
-        >>> sq.isActive
+
+        >>> sq.isActive # doctest: +SKIP
```

Comment:
is this required? Why is it skipped?

Reply:
Calling isActive after the query is stopped will throw error right now before the better session management is implemented

Reply:
Is see, please add a TODO here with a brief comment.